src/overlays/actors/ovl_En_Az/z_en_az.c

```diff
@@ -5,28 +5,72 @@
  */
 
 #include "z_en_az.h"
+#include "assets/objects/object_az/object_az.h"
+#include "overlays/actors/ovl_En_Twig/z_en_twig.h"
+#include "overlays/actors/ovl_En_Fish/z_en_fish.h"
 
 #define FLAGS (ACTOR_FLAG_10 | ACTOR_FLAG_80000000)
 
 #define THIS ((EnAz*)thisx)
 
+#define GET_PARAM1(beaver) (((beaver)->params >> 8) & 0xF)
+#define GET_PARAM2(beaver) ((beaver)->params & 0xFF)
```

Comment:
You'll want to move these to the header file. Also, it should take `thisx` I believe